readthedocs/projects/models.py

```diff
@@ -409,11 +409,14 @@ def get_production_media_path(self, type_, version_slug, include_file=True):
 
     def get_production_media_url(self, type_, version_slug, full_path=True):
         """Get the URL for downloading a specific media file."""
-        path = reverse('project_download_media', kwargs={
-            'project_slug': self.slug,
-            'type_': type_,
-            'version_slug': version_slug,
-        })
+        try:
+            path = reverse('project_download_media', kwargs={
+                'project_slug': self.slug,
+                'type_': type_,
+                'version_slug': version_slug,
+            })
+        except NoReverseMatch:
+            return ''
```

Comment:
Looks like this is only used in `Project.get_downloads`, perhaps that should receive the exception and abort building a list of downloads in that case?

That might bleed into passing that to the flyout though, I'm not sure if there is anything expecting for the keys in `downloads` to exist.